RobotFramework_Testsuites/Utils/LibListener.py

```diff
@@ -81,6 +82,16 @@ def _start_suite(self, data, result):
             RobotFramework_Testsuites.CTestsuitesCfg.oConfig.sRootSuiteName = test_suite.name
             RobotFramework_Testsuites.CTestsuitesCfg.oConfig.iTotalTestcases = test_suite.test_count
             
+            if '${localconfig}' in BuiltIn().get_variables()._keys:
+                if re.match('^\s*$', BuiltIn().get_variable_value('${LOCAL_CONFIG}')):
+                    logger.error("local_config input must not be empty!!!")
+                else:
+                    RobotFramework_Testsuites.CTestsuitesCfg.oConfig.lLocalConfig.append(BuiltIn().get_variable_value('${LOCAL_CONFIG}').strip())
+            elif os.path.isdir(os.environ['ROBOT_LOCAL_CONFIG']):
```

Comment:
Hi Son,
if have concerns if it is useful to load config files from a whole directory.
This is not required, especially due to our json-preprocessor which allows importing of files.
So, please remove this part. This makes configuration scenarios too complex.

ROBOT_LOCAL_CONFIG could be a specific file. This makes sense, but then commandline must still use the one provided by command line instead of ROBOT_LOCAL_CONFIG.
This can be maybe useful for CI jobs.

Please implement accordingly.

Thank you,
 Thomas